### ISSUE TITLE
Doc: should style of <Line> be 'stroke' instead of 'fill'

### ISSUE THREAD

**Issue by lrrfantasy** [zcreativelabs/react-simple-maps#88]
Hey,

I'm following the doc of https://github.com/zcreativelabs/react-simple-maps#Line-component as there is no example of Line.

My code looks as
```jsx
<Line line={{ coordinates: {
    start: [139.683, 35.683],
    end: [0, 0]
  } }} style={{
    default: { fill: "#FF5722" },
    hover: { fill: "#FFFFFF" },
    pressed: { fill: "#FF5722" },
  }}
/>
```

The line doesn't show as it's not an enclosed shape, 'fill' here doesn't render the line. When replacing 'fill' with 'stroke' in style, the line renders. This appears the same as using `path` directly ('fill' doesn't render straight line).

I wonder if it's a doc mistake where `stroke` should be used. An example of usage of `<Line/>` would be much more helpful

**Comment by zimrick** [comment]
Hi @lrrfantasy,

You are absolutely right, that should be `stroke` instead of `fill`. Thank you for spotting this!

**Comment by zimrick** [comment]
Fixed c60cb0b35985eb739f50461c73ee079c04929570.
